Library/Homebrew/extend/ENV/super.rb

```diff
@@ -331,6 +334,12 @@ def libcxx
     append_to_cccfg "g" if compiler == :clang
   end
 
+  sig { void }
+  def debug_symbols
```

Comment:
@carlocab would love some help on how to connect this to the `--debug-symbols` flag. I don't see a clear path to follow right now (as there are no flags that affect compilation today).

Reply:
> as there are no flags that affect compilation today

That's not quite true; the `--cc` flag affects compilation. You'll probably need to set this (perhaps with `ENV.debug_symbols`, though there may be a cleaner way of doing it) inside `install` in `build.rb`.